src/Utils/isAssignableTo.ts

```diff
@@ -69,6 +70,8 @@ function getPrimitiveType(value: LiteralValue) {
             return new NumberType();
         case "boolean":
             return new BooleanType();
+        case "symbol":
+            return new SymbolType();
```

Comment:
This line wasn't touched. Can you extend a test case or add a test case?

Reply:
Looks like I was being a little overzealous with this addition as symbols literals aren't fully supported in TypeScript yet. It also looks like there would be considerably more work to get it playing right with the parser.